pkg/agent/core/goalcu.go

```diff
@@ -120,6 +136,6 @@ func calculateLFCGoalCU(
 			return nil
 		}
 
-		return lfcGoalCU, lfcLogFields
+		return lfcGoalCU, lo.ToPtr(api.Bytes(estimateWssMem)), lfcLogFields
```

Comment:
Perhaps split this function into two: one calculates estimateWssMem, the other turns it into lfcGoalCU?

Reply:
I started writing this, but the handling with log fields gets tricky when it's split across multiple functions... WDYT?

Reply:
No problem, let's leave as-is.